airbyte-integrations/connectors/source-freshdesk/source_freshdesk/schemas/groups.json

```diff
@@ -30,7 +30,7 @@
       "type": "string"
     },
     "auto_ticket_assign": {
-      "type": ["string", "null"]
+      "type": ["integer", "null"]
```

Comment:
just curious, how did you find these schema issues?

Reply:
So the `test_core.test_read()` test has a validate schema check which was failing during the SAT run above. So then just to double check I made the requests against the freshdesk API in postman and updated all the types accordingly.